apps/heft/src/schemas/heft.schema.json

```diff
@@ -108,6 +108,34 @@
       "items": { "$ref": "#/definitions/heft-plugin" }
     },
 
+    "aliasesByName": {
+      "type": "object",
+      "description": "Defines aliases for existing Heft actions, and allows them to be invoked by name with default parameters.",
+      "additionalProperties": false,
+      "patternProperties": {
+        "^[a-z][a-z0-9]*([-][a-z0-9]+)*$": {
+          "description": "Defines a Heft action alias.",
+          "type": "object",
+          "additionalProperties": false,
+          "required": ["actionName"],
+          "properties": {
+            "actionName": {
+              "description": "The name of the Heft action to invoke.",
+              "type": "string",
+              "pattern": "^[a-z][a-z0-9]*([-][a-z0-9]+)*$"
+            },
+            "defaultArguments": {
```

Comment:
In the terminology of `ts-command-line`:

If you write `--to-except PHASE` for example `--to-except build`:
- The `--to-except` part is called the **parameter**
- The `build` part is called the **argument** and `PHASE` is the **argument name**.

Reply:
So I think this should be called `defaultParameters`?

Reply:
It can be named parameters... the reason I stuck with arguments is because it's a raw string array of args that get passed through on the CLI.

Reply:
`defaultArgv`?